site/lib/mergeHoldings.test.ts

```diff
@@ -0,0 +1,406 @@
+import { Holding } from "components/pages/Pledge/types";
+import {
+  calculateChange,
+  calculateTokenValues,
+  groupHoldings,
+  mergeHoldings,
+} from "./mergeHoldings";
+
+const holdingsInput: Holding[][] = [
+  [
+    {
+      id: "0xdeb8c24ad9640d62334da6f30c191e9cfd180e5b-BCT-1674777600",
+      timestamp: "1",
+      token: "BCT",
+      tokenAmount: "1",
+      carbonValue: "1",
+    },
+    {
+      id: "0xdeb8c24ad9640d62334da6f30c191e9cfd180e5b-BCT-1674777600",
```

Comment:
we actually don't care about `id` or `carbonValue` so I would remove it from the fixtures for readability. You can do typescript asserts if needed (avoid `any` though b.c. our linter hates it)
```ts
mergeHoldings(myFixture as Holding[])
```

Reply:
@Atmosfearful I actually found the ids useful when looking through the test features because they are the wallet address. This makes it easier to look at the giant arrays and see how the data is being merged. I can trim off the token and timestamp at the end to make it shorter or change it to something like wallet1 and wallet2. carbonValue removed for inputs but `calculateTokenValues` returns carbonValue so test fails if its not included in expected output.